src/Microsoft.DocAsCode.Build.Engine/PostProcessors/IncrementalPostProcessorHost.cs

```diff
@@ -29,8 +30,16 @@ public IncrementalPostProcessorHost(IncrementalPostProcessorsContext increContex
 
             _increContext = increContext;
             _postProcessorName = postProcessorName;
+            SourceFileInfos = sourceFileInfos;
```

Comment:
offline discussed, outputfile  may be mofidied by one of the post processors, so pass sourceFileInfo to post processors, post processor to handle the mapping from source to output.

---
In reply to: [97193321](https://github.com/dotnet/docfx/pull/1208#discussion_r97193321) [](ancestors = 97193321)